librosa/filters.py

```diff
@@ -642,7 +643,7 @@ def constant_q_lengths(sr, fmin, n_bins=84, bins_per_octave=12,
         raise ParameterError('Filter pass-band lies beyond Nyquist')
 
     # Convert frequencies to filter lengths
-    lengths = Q * sr / freq
+    lengths = Q * sr / (freq + gamma)
```

Comment:
I do not think this is the correct way to calculate the lengths.

The paper defines the bandwidth as: alpha * f_k + gamma,
where we can think of alpha as the inverse of Q (my understanding is that it is arbitrary which convention is used, but I could be wrong - it could also affect reconstruction).

This code currently implements: alpha * (f_k + gamma)

Reply:
Ah good catch!  I believe you're correct -- must have gotten messed up in the translation from their notation to ours.